src/main/java/sirius/biz/protocol/Protocols.java

```diff
@@ -140,6 +145,47 @@ public void handle(Incident incident) throws Exception {
         }
     }
 
+    private String truncateStacktrace(Incident incident) {
+        boolean foundStackTrace = false;
+        StringBuilder result = new StringBuilder();
+        StringBuilder stack = new StringBuilder();
+        StringBuilder errorMesssage = new StringBuilder();
+
+        for(String line: NLS.toUserString(incident.getException()).split("\\r?\\n")) {
+            if (!foundStackTrace) {
+                if (line.contains("at") && line.contains(".java")) {
```

Comment:
Don't u wanna add white spaces around "at", since you are probably looking for `something at somewhere`?

Reply:
![Bildschirmfoto 2024-05-15 um 12 54 03](https://github.com/scireum/sirius-biz/assets/106659278/99d1597e-c7e2-4586-94f6-1dd521844c67)

Nope. I am trying to find every line of the original error message in case that the error message has several new line chars.
Therefore I am looking for the first line that contains an "at" and an ".java" as these lines mark the start of the actual stacktrace.

Reply:
![Bildschirmfoto 2024-05-15 um 12 55 56](https://github.com/scireum/sirius-biz/assets/106659278/8fb98ced-63ad-4daa-9b32-346f51a7e789)

Then I am trying to add up the stacktrace until the occurence of a "Caused by:" as the error message might be repeated here. Then I repeat the algorithm to find the error message and the stacktrace of the cause.

Reply:
I agree with @idlira. The check seems to be a bit too sketchy to me. You could check whether you have whitespace, the literal `at`, more whitespace, a Java symbol name, and then, in brackets, a file name and a line number. While a full check may be too much, your check does not really do justice to the regular pattern of such a stack-trace line